policyengine_us/variables/gov/states/wv/tax/income/subtractions/wv_social_security_benefits_subtraction.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class wv_social_security_benefits_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "West Virginia social security benefits subtraction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        # West Virginia Personal Income Tax Forms And Instructions 2020 LINE 33
+        "https://tax.wv.gov/Documents/TaxForms/2020/it140.booklet.pdf#page=24",
+        # West Virginia Personal Income Tax Forms And Instructions 2020 LINE 32
+        "https://tax.wv.gov/Documents/TaxForms/2021/it140.booklet.pdf#page=24",
+        # West Virginia Personal Income Tax Forms And Instructions 2022 LINE 32
+        "https://tax.wv.gov/Documents/PIT/2022/PersonalIncomeTaxFormsAndInstructions.2022.pdf#page=25",
+        # Code of West Virginia §11-21-12 (c)(8)(A) - (c)(8)(C)
+        "https://code.wvlegislature.gov/11-21-12/",
+    )
+    defined_for = "wv_social_security_benefits_subtraction_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.wv.tax.income.subtractions.social_security_benefits
+        person = tax_unit.members
+
+        head_or_spouse = person("is_tax_unit_head_or_spouse", period)
+        social_security = tax_unit.sum(
```

Comment:
please line split this comment. also for future reference, `add(person, period, ["taxable_social_security", "ssi"])` could have been more efficient for the previous logic
```suggestion
        # "social security benefits paid by the Social Security Administration as Old Age, Survivors and Disability Insurance Benefits as provided in §42 U.S.C. 401 et. seq. or as Supplemental Security Income for the Aged, Blind, and Disabled as provided in §42 U.S.C. 1381 et. seq., included in federal adjusted gross income"
        # SSI is not included in federal AGI, so limit to taxable SS.
        taxable_ss = person("taxable_social_security", period)
        total_eligible_ss = tax_unit.sum(taxable_ss * head_or_spouse)
        return total_eligible_ss * p.rate
```

Reply:
Please do not resolve a suggestion comment without accepting it or explaining why the suggestion is incorrect

Reply:
Sorry, I'm not sure if you were commenting on a outdated commit. I've modified the code according to the suggestion already.